.github/workflows/storybook.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Deploy
+on:
+  push:
```

Comment:
Let's leave it to all pushes for now, but we will probably prefer deploying docs only upon release so the docs always corresponds to the latest version published.

Reply:
For testing deploy I switched it to all pushes for now. However I want to deploy storybook only on push to main branch and also `paths` option sounds good to me.

Reply:
OK, let's have it this way now.

Reply:
Changed to pushes only to the `main` branch.